sections/getting-started.md

```diff
@@ -0,0 +1,156 @@
+# Getting Started
+
+## Prerequisites
+
+1. A [Google Ad Manager](https://admanager.google.com/intl/en_uk/home/) account
+1. A working understanding of Google Ad Manager (their [Ad selection whitepaper](https://support.google.com/admanager/answer/1143651?hl=en) is a good place to start)
+1. A working understanding of [Google Publisher Tag](https://developers.google.com/publisher-tag/guides/get-started)
+1. A working understanding of your site's header bidding configuration (ie [Prebid.js](https://docs.prebid.org/), [APS](https://aps.amazon.com/aps/index.html), etc)
+1. A Glimpse Zero account ([contact us if you don't have one](mailto:support@glimpsezero.io))
+
+## Basic setup
+
+There are three steps to get up and running:
+
+1. Integrating our library on your page (only done once)
+1. Setting up your Sponsorship Line Item in GAM (done for each Sponsorship)
+1. Setting up targetting in Glimpse Zero (done for each Sponsorship)
+
+### Page integration
+
+1. Include the zero.js tag in `<head>` tags:
+
+  ```html
+  <script async src="https://cdn.glimpsezero.io/scripts/zero.js"></script>
+  ```
+
+  > ℹ️ You can fix the zero.js library to a specific version by appending the SEMVER to src url: `zero-1.0.1.js`. Alternatively if you use `zero.js` the latest version is used.
+
+1. Initialised the zero.js command queue:
+
+  ```ts
+  var zjs = window.zjs || {}
+  zjs.cmd = window.zjs.cmd || []
+  ```
+
+1. Set your 4 character publisher id:
+
+  ```ts
+  zjs.setPubId("a9j3")
+  ```
+
+1. Finally run the prematch process and pass the unmatched ad units to prebid:
```

Comment:
same comment: replace "prebid" with "usual ad calls" / "usual demand sources" or similar?

Reply:
I've replaced prebid with "header bidding setup" ... i think this should be broad enough to capture APS (amazon pub services which includes TAM/UAM). But we can confirm with Jordan to make sure the language is right.